shared/wallets/wallet/settings/popups/really-remove-account/container.js

```diff
@@ -28,12 +29,16 @@ const mapDispatchToProps = (dispatch, {navigateUp}) => ({
     dispatch(navigateUp())
   },
   _onCopyKey: (secretKey: string) => dispatch(ConfigGen.createCopyToClipboard({text: secretKey})),
-  _onFinish: (accountID: Types.AccountID) =>
+  _onFinish: (accountID: Types.AccountID) => {
     dispatch(
       WalletsGen.createDeleteAccount({
         accountID,
       })
-    ),
+    )
```

Comment:
Also, clicking the 'x' on this dialog brings us back to the remove-account screen, both on old nav and new nav. Is that intended?

Reply:
i think it can go all the way back, whatever feels more natural

Reply:
I left this as-is for now.